### ISSUE TITLE
support hoisted node_modules

### ISSUE THREAD

**Issue by Bkucera** [mgol/check-dependencies#36]
right now it only searches the sibling `node_modules` folder, but it would be nice to search `node_modules` that may be hoisted up further in the tree, since `node` will be able to resolve them.

e.g.:
```py
root
	package.json
	node_modules/
		a-module-dep
	packages/
		a-module
		package.json
```

this way I could use `yarn workspaces`, which hoists node_modules of subpackages in a monorepo

**Comment by Raag007** [comment]
Hello Everyone, 
Can we please get support for this. 

We have packages structure like below: 
```
Lerna-Repo
node_modules
package.json 
        package1
                package.json
                node_modules
        package2
                package.json
                node_modules
```
We are using `lerna bootstrap --hoist`
It will be so helpful if the tool search `node_modules` that are hoisted up to the root.

cc: @Bkucera 


Thank you in advance.

**Comment by mgol** [comment]
I agree it’d be useful in some cases. But I don’t have time to do it myself at the moment; PRs (with tests) welcome.

**Comment by isoroka-plana** [comment]
Ran into the same issue with npm workspaces. Looks like I cannot adapt this package in my monorepo at the moment.
